soc/ite/ec/it51xxx/chip_chipregs.h

```diff
@@ -339,6 +339,24 @@ struct gctrl_it51xxx_regs {
 #define gctrl_ite_ec_regs      gctrl_it51xxx_regs
 #define GCTRL_ITE_EC_REGS_BASE GCTRL_IT51XXX_REGS_BASE
 
+/**
+ *
+ * (22xxh) Battery-backed SRAM (BRAM) registers
+ *
+ */
+#ifndef __ASSEMBLER__
+/* Battery backed RAM indices. */
+#define BRAM_MAGIC_FIELD_OFFSET 0x7c
+enum bram_indices {
+
```

Comment:
nit: drop this blank line? if anything it should go before the enum (move up one line)

Reply:
Done.